src/ducks/Signals/signalFormManager/signalCrudForm/signal/TriggerForm.module.scss

```diff
@@ -124,17 +124,33 @@ label {
 .timeWindow {
   display: flex;
   align-items: center;
-}
 
-.timeWindowInput {
-  flex: 1;
-  width: 50%;
-  margin-right: 8px;
+  .timeWindowInput {
+    flex: 1;
+    width: 50%;
+    margin-right: 8px;
+  }
+
+  .timeWindowUnit {
+    width: 50%;
+    flex: 1;
+  }
 }
 
-.timeWindowUnit {
-  width: 50%;
-  flex: 1;
+.frequency {
+  display: flex;
+  align-items: center;
+
+  .frequencyTimeValue {
+    flex: 1;
+    width: 50%;
+    margin-right: 8px;
+  }
+
+  .frequencyTimeType {
```

Comment:
Why the selectors on LOC `128`, `134`, `144`,` 150` are nested? Is there a need for this kind of specificity?

Reply:
Because element depends from their parents. DOM tree like scss tree.
It it is important I can revert this changes.

Reply:
Why generate `.frequency .frequencyTimeValue {}` when it's possible generate `.frequencyTimeValue {}`?